view/rooms/list/app.js

```diff
@@ -29,7 +29,7 @@ window.onload = () => {
   let rooms;
   let roomTypes;
   uPage.setRoomsTab();
-  u.getPromiseWithUrl(GET_ROOMS)
+  u.getPromiseWithUrl(`${GET_ROOMS}?sort=-id`)
```

Comment:
Just to confirm, you want to sort it by `name` or `id` ?

Reply:
@shriramshankar yeah rooms ids are integers so the higher the room number the newer the room